composites/Plugin/SnippetPreview/components/HelpTextWrapper.js

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import HelpText from "../../Shared/components/HelpText";
+import colors from "../../../../style-guide/colors.json";
+import { Button } from "../../Shared/components/Button";
+import SvgIcon from "../../Shared/components/SvgIcon";
+import { rgba } from "../../../../style-guide/helpers";
+
+
+const HelpTextContainer = styled.div`
+	max-width: 600px;
+    padding: 1 0 3 1em;
+    font-weight: normal;
+    margin: 0px 20px 10px 25px
+`;
+
+let HelpTextDiv = styled.div`
+	max-width: 400px;
+	display: block;
+	transition: all 0.5s ease;
+	overflow: hidden;
+	max-height: 0;
+`;
+
+const HelpTextButton = styled( Button )`
+	min-width: 14px;
+	min-height: 14px;
+	width: 30px;
+	height: 30px;
+	border-radius: 50%;
+	border: none;
+	clip: rect(1px 1px 1px 1px);
+	box-shadow: 0 0 0 0px #5b9dd9;
```

Comment:
0px to 0, is there a color in colors.json that corresponds to the box shadow color?